apps/test/unit/templates/AgeDialogTest.js

```diff
@@ -1,28 +1,40 @@
-import {assert} from '../../util/deprecatedChai';
+import {assert} from '../../util/reconfiguredChai';
```

Comment:
Maybe better, because we only use vanilla chai features in this test?
```suggestion
import {assert} from 'chai';
```